### PR TITLE
fix for deprecated code

### PR DESCRIPTION
**Pull request by katotekii**
## Checklist

- using ox lib context menu instead of qb menu

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -497,12 +497,29 @@ end)
 
 RegisterNetEvent('qb-garbagejob:client:MainMenu', function()
     local MainMenu = {}
-    MainMenu[#MainMenu+1] = {isMenuHeader = true,header = Lang:t("menu.header")}
-    MainMenu[#MainMenu+1] = { header = Lang:t("menu.collect"),txt = Lang:t("menu.return_collect"),params = { event = 'qb-garbagejob:client:RequestPaycheck',}}
```
Comment (BerkieBb) [inline client/main.lua:501]: Let's keep this line instead of adding it to an else statement and indenting it

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 1 above)
Comment (katotekii) [reply client/main.lua:501]: do whatever you think it should be, I just provided working solution :D

Comment (BerkieBb) [reply client/main.lua:501]: It's basically just moving one line back here

Comment (BerkieBb) [reply client/main.lua:501]: So just take
```lua
        MainMenu[#MainMenu + 1] = {
            title = Lang:t("menu.collect"),
            description = Lang:t("menu.return_collect"),
            event = 'qb-garbagejob:client:RequestPaycheck'
        }
```
Out of the if statement and put it above, then it's already good

**Comment by ChatDisabled** [comment]
Same story: https://github.com/Qbox-project/qbx-garbagejob/commit/dff89bf481c2b37fd470021d4e1adfa25c1db355

Thanks @katotekii ❤️

**Pull request closed without merging**